client/src/components/ModalComponents/DatepickerModal/DatepickerModal.tsx

```diff
@@ -31,6 +51,8 @@ const DatepickerModal = () => {
             onChange={(date) => {
               setStartDate(date);
               console.log(date);
+              date ? setSelectedDate(date) : null;
```

Comment:
삼항식보다는 if문으로 표현해주시면 더 좋을 것 같습니다~
```suggestion
             if(date) {
               setSelectedDate(date)
             }
```